### PR TITLE
feat: List shipping options for cart

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]

File: integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts [hunk 1]
```diff
@@ -672,12 +678,84 @@ medusaIntegrationTestRunner({
             expect(updatedItem).not.toBeUndefined()
           })
         })
+      })
 
-        describe("createPaymentCollectionForCart", () => {
-          it("should create a payment collection and link it to cart", async () => {
-            const cart = await cartModuleService.create({
-              currency_code: "dkk",
+      describe("createPaymentCollectionForCart", () => {
```
Comment (olivermrbl) [inline integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts:686]: comment: This is just a rearrangment. It was placed inside an incorrect describe.

File: integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts [hunk 2]
```diff
@@ -63,6 +66,9 @@ medusaIntegrationTestRunner({
         fulfillmentModule = appContainer.resolve(
           ModuleRegistrationName.FULFILLMENT
         )
+        locationModule = appContainer.resolve(
+          ModuleRegistrationName.STOCK_LOCATION
+        )
         remoteLink = appContainer.resolve("remoteLink")
         remoteQuery = appContainer.resolve("remoteQuery")
```
Comment (adrien2p) [inline integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts]: nit: we can use the container registration name to resolve those

File: integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts [hunk 3]
```diff
@@ -1077,6 +1083,337 @@ medusaIntegrationTestRunner({
...
+          const serviceZone = await fulfillmentModule.createServiceZones({
+            name: "Test",
+            fulfillment_set_id: fulfillmentSet.id,
+            geo_zones: [
+              {
+                type: "country",
+                country_code: "us",
+              },
+            ],
+          })
```
Comment (adrien2p) [inline integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts]: suggestion: you can do all that using the create only and it should be enough

File: integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts [hunk 4]
```diff
@@ -1077,6 +1083,337 @@ medusaIntegrationTestRunner({
...
```
Comment (adrien2p) [inline integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts]: same suggestion as above

**Review by adrien2p — APPROVED** [review]
🔥

**Review by sradevski — APPROVED** [review]

**Pull request merged**